src/sender.js

```diff
@@ -101,7 +110,16 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue }) {
         console.error(`Tx Failed for event Tx ${job.transactionReference}`)
         failedTx.push(job)
 
-        if (
+        if (e.message.includes('Insufficient funds')) {
+          insufficientFunds = true
+          const currentBalance = await web3Instance.eth.getBalance(VALIDATOR_ADDRESS)
+          minimumBalance = toBN(job.gasEstimate)
+            .add(toBN(200000))
```

Comment:
Could you clarify why you added 200_000 of gas? Does it make sense to add 20%?
Whatever the value you are using, let use a constant and provide a comment to expose its meaning. Thanks

Reply:
Oh, yes, sorry. That is the amount that is being added some lines above to the estimated gas. But I totally agree that it should be in a constant. Maybe something like `EXTRA_GAS = 200000`? Or we could use a percentage.

Reply:
I would prefer to use a percentage. Otherwise for transactions which requires less than 100 thousands the gas parameter will have the value 3 times more than required.

Reply:
Done. I added a constant for the extra percentage and a function that applies this increase.